samples/react/README.md

```diff
@@ -0,0 +1,85 @@
+This project integrates monaco-kusto in a create-react-app application.
+
+The following changes were made to a default create-react-app application:
+
+1. Used script tags to import Kusto language service (this piece of code is C# code transpiled to javascript. it has several issues that prevent it from being imported as a regular import statement).
+
+2. Added a prepare task to package.json that copies over monaco and monaco-kusto to the public folder (this is done because we're using UMD to require monaco and monaco-kusto in runtime. otherwise, we would have to eject create-react-app and play with webpack config as explained [here](https://github.com/microsoft/monaco-editor/blob/master/docs/integrate-esm.md) )
+
+3. Added monaco-kusto.js which takes care of loading monaco (by using @monaco-editor/react package) and then loading monaco-kusto
+
+4. Added KustoEditor.jsx which is a react component that uses monaco-kusto to initialize monaco + monaco-kusto and then creates an editor configure for the kusto language. It then pushes a static cluster schema (which it takes from schema.js).
+
+In real production apps, at least the following changes will have to be made:
+
+1. KustoEditor will have to contain some props to customize the initialization of monaco-editor
+2. the app will use '.show schema as json' to fetch the schema from a real Kusto cluster.
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `yarn start`
```

Comment:
After running yarn and yarn start I got this error (tried the steps but got the same error):
![image](https://user-images.githubusercontent.com/8294298/96571838-f7f3dc80-1280-11eb-9885-5d1701d66206.png)

Reply:
yes